### PR TITLE
MOSIP-31683 added logger for debugging

### PR DESCRIPTION
**Pull request by Neha2365**

### CODE REVIEW

**Review by vadherhemant — APPROVED** [review]
approved for debuggin.  needs to be reverted before the release or after debuggin is completed.

File: id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java [hunk 1]
```diff
@@ -163,6 +163,11 @@ public ResponseWrapper<CredentialIssueResponse> createCredentialIssuanceByRid(Cr
 
 		CredentialIssueResponse credentialIssueResponse = null;
 		try{
+			Optional<CredentialEntity> entity = credentialDao.findById(rid);
+			if (entity.isPresent()) {
+				LOGGER.debug(IdRepoSecurityManager.getUser(), CREDENTIAL_SERVICE, CREATE_CREDENTIAL,
```
Comment (LoganathanSekar7627) [inline id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java:168]: @Neha2365 , you will need to use + to append multiple texts. if you use , it is expected that the first text is having placeholder (%s) for the next following arguments.

Comment (LoganathanSekar7627) [reply id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java:168]: Also please add a logger for else block also.

**Review by LoganathanSekar7627 — CHANGES_REQUESTED** [review]

**Review by LoganathanSekar7627 — APPROVED** [review]

**Comment by Neha2365** [comment]
Not required  these changes

**Pull request closed without merging**